src/vs/platform/quickinput/browser/pickerQuickAccess.ts

```diff
@@ -326,6 +326,14 @@ export abstract class PickerQuickAccessProvider<T extends IPickerQuickAccessItem
 
 		// Accept the pick on accept and hide picker
 		disposables.add(picker.onDidAccept(event => {
+			if (runOptions?.handleAccept) {
```

Comment:
Can't you use the 'accept' callback on the item to do the same thing? I don't follow

Reply:
We don't control the items in the picker, they are populated from the anything quick access in this case

Reply:
We could overwrite the `accept` handler for all items in the anything quick access layer, but this feels more explicit